src/lib/Scenes/Search/ResultWithHighlight.tsx

```diff
@@ -0,0 +1,141 @@
+import GraphemeSplitter from "grapheme-splitter"
+import { normalizeText } from "lib/utils/normalizeText"
+import { Text as PaletteText, TextProps } from "palette"
+import React from "react"
+
+const splitter = new GraphemeSplitter()
+
+export const Text: React.FC<TextProps> = ({ children, ...rest }) => {
+  return (
+    <PaletteText variant="xs" {...rest}>
+      {children}
+    </PaletteText>
+  )
+}
+
+export const ItalicText: React.FC<{ color?: string }> = ({ color = "grey", children }) => {
```

Comment:
I think that this should be black60 according to figma

```suggestion
export const ItalicText: React.FC<{ color?: string }> = ({ color = "black60", children }) => {
```